Moblin/Various/Model.swift

```diff
@@ -1197,20 +1198,23 @@ final class Model: NSObject, ObservableObject {
             if self.database.debug!.enableRtmpAudio! {
                 self.media.addRtmpAudio(cameraId: stream.id)
             }
+            stream.isConnected = true
```

Comment:
Hmm, is it really necesary to keep track of the RTMP stream status in the stream "settings". I mean, isn't it possible to call `isRtmpStreamConnected()` on the model somehow?

I have not looked into the details though, but prefer keeping the connection state in one place.

Reply:
cool, I didn't discover the function. That is of course much better. will adjust it accordingly.

Reply:
I hope it works as I didn't look too closely at it. Too much in Moblin, so hard to remember everything =)